### PR TITLE
test case for #23

### PR DESCRIPTION
**Pull request by cdietrich**
test case for #23
Signed-off-by: Christian Dietrich <christian.dietrich@itemis.de>

### CODE REVIEW

File: tycho-its/projects/issue23/bundle/build.properties [hunk 1]
```diff
@@ -0,0 +1,5 @@
+source.. = src/
+output.. = bin/
+bin.includes = .,\
+               META-INF/
+additional.bundles = org.eclipse.emf.codegen.ecore
```
Comment (cdietrich) [inline tycho-its/projects/issue23/bundle/build.properties:5]: unfortunately the problem is not reproducible with a dependency closer to platform

File: tycho-its/projects/issue23/bundle/.gitignore [hunk 2]
```diff
@@ -0,0 +1,2 @@
+/target
```
Comment (laeubi) [inline tycho-its/projects/issue23/bundle/.gitignore:1]: itest projects should not contain individual git ignores

File: tycho-its/projects/issue23/bundle/build.properties (hunk 1 above)
Comment (laeubi) [reply tycho-its/projects/issue23/bundle/build.properties:5]: I found org.junit useful for this purpose as it is part of the sdk but I think it does not mater much as long as we do not need to include large dependency chains.

**Review by laeubi — CHANGES_REQUESTED** [review]
Thanks for the test-case, just a minor adjustment (git ignore). BTW you don't need to force-push, just push to your branch and we can later squash it.

**Comment by cdietrich** [comment]
reworked to have two bundles, and use junit as add. bundle in the first

**Review by laeubi — COMMENTED** [review]
.project files and .classpath files are not necessary for this also m2e.core prefs can be omited

**Review by laeubi — APPROVED** [review]
Looks good to me, if build succeeds we can merge this

**Pull request merged**